docs/cloudformation.json

```diff
@@ -298,6 +298,13 @@
                 "ec2:Describe*",
                 "elasticloadbalancing:*",
                 "ecs:*",
+                "ecr:GetAuthorizationToken",
+                "ecr:BatchCheckLayerAvailability",
```

Comment:
Minor but, `ecr:GetAuthorizationToken` looks like it should be the only permission Empire needs?